frontend/src/components/Item/Identify/IdentifyResult.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-card
+    class="identify-result-block"
+    variant="outlined"
+    @click="$emit('select', item)">
+    <poster-block :url="item.ImageUrl ?? undefined" :item-type="itemType" />
+    <div class="text-center text-subtitle-1 font-weight-bold mt-2 mx-2">
+      {{ item.Name }}
+    </div>
+    <div
+      class="text-center text-body-2 mt-1 mx-2"
+      :class="{
+        'mb-2': typeof item.ProductionYear !== 'number'
```

Comment:
I don't understand exactly why this style change is needed? 😅
Can you add some screenshots please?

Reply:
Since ProductionYear can be missing, we don't want to add extra bottom margin between it. Although I guess it's not as readable as just using `my-2` in the ProductionYear <div/>